plugin/NERD_commenter.vim

```diff
@@ -450,6 +450,7 @@ let s:delimiterMap = {
     \ 'typescript': { 'left': '//', 'leftAlt': '/*', 'rightAlt': '*/' },
     \ 'typescriptreact': { 'left': '//', 'leftAlt': '{/*', 'rightAlt': '*/}' },
     \ 'uc': { 'left': '//', 'leftAlt': '/*', 'rightAlt': '*/' },
+    \ 'uc4': { 'left': '! ', 'leftAlt': '!' },
```

Comment:
Please drop the space from the base delimiter (and alternate). I understand the space will be proffered by many/most programmers (myself included), but if it isn't mandated my the language parser then we're going to leave that up to the user's preferences — for which there is a setting. I'm happy to merge the additionally language support, but for the special space treatment you should either use the space-delims setting or override this one language in your own RC file.